src/NextLink.css

```diff
@@ -0,0 +1,5 @@
+.link {
```

Comment:
This class `link` has a great probability of creating collisions with the css of someones project.

Me and @PedroMiguelSS are trying to figure a good way to avoid this. If you guys have any suggestions, please share 🙏

Reply:
a few suggestions:

- `.moxy-link`
- `.moxy-next-link`
- `.m-link`
- `.m-next-link`

Reply:
I agree with you @zebateira. I would like to add one option. 

Every class or id could have a prefix. The prefix would be the initials of the package. For example:

(next-link)
- `nl_link`
- `nl_container`

This is very similar to what we have in react-carousel (`rc`)

Reply:
I like `moxy-next-link`, although verbose it diminishes the chance of collision, while `nl_link` is still a bit more broad.

I'll go with the `moxy-next-link` for now 👍

Reply:
After talking with @PedroMiguelSS the above solution might not be great, since we may have collisions with other repos. For example:
- next-link (nl_container)
- next-label (nl_container)

So, I would actually be more verbose just to make sure we avoid this as much as possible :+1: